RealmSwift/RealmCollection.swift

```diff
@@ -1190,3 +1215,14 @@ extension LinkingObjects: ObservableCollection {
         return objc === rlmResults
     }
 }
+
+// MARK: Key Path Strings
+
+/// Tag protocol which allows a collection to produce its property name
+internal protocol KeyPathStringCollection {
```

Comment:
This protocol needs an adjective as it makes little sense otherwise. I would call it `PropertyNameDiscoverable` or `PropertyNameConvertible`.

Reply:
changed to PropertyNameConvertible